src/main/java/com/onarandombox/MultiverseCore/commands/GameruleCommand.java

```diff
@@ -64,17 +65,38 @@ public void runCommand(CommandSender sender, List<String> args) {
             world = Bukkit.getWorld(args.get(2));
             if (world == null) {
                 sender.sendMessage(ChatColor.RED + "Failure!" + ChatColor.WHITE + " World " + ChatColor.AQUA + args.get(2)
-                    + ChatColor.WHITE + " does not exist.");
+                        + ChatColor.WHITE + " does not exist.");
                 return;
             }
         }
 
-        if (world.setGameRuleValue(gameRule, value)) {
-            sender.sendMessage(ChatColor.GREEN + "Success!" + ChatColor.WHITE + " Gamerule " + ChatColor.AQUA + gameRule
-                    + ChatColor.WHITE + " was set to " + ChatColor.GREEN + value);
+        if (gameRule == null) {
+            sender.sendMessage(ChatColor.RED + "Failure! " + ChatColor.AQUA + args.get(0) + ChatColor.WHITE
+                    + " is not a valid gamerule.");
         } else {
-            sender.sendMessage(ChatColor.RED + "Failure!" + ChatColor.WHITE + " Gamerule " + ChatColor.AQUA + gameRule
-                    + ChatColor.WHITE + " cannot be set to " + ChatColor.RED + value);
+            boolean success = false;
+
+            try {
+                if (gameRule.getType() == Boolean.class) {
+                    if (value.equalsIgnoreCase("true")) success = world.setGameRule(gameRule, true);
+                    else if (value.equalsIgnoreCase("false")) success = world.setGameRule(gameRule, false);
```

Comment:
Not a big fan of single line if statements unless it is only if (no else) and is VERY simple. Definitely no assignment. Please use braces here. Also, looks like we should probably consider the `else` case here too.

Reply:
Absolutely, will get braces in there. Also regarding the `else` case, see below on the flag.

Reply:
Right, but couldn't the message here indicate that true or false is expected?

Reply:
Done.